.gitlab/source_test/ebpf.yml

```diff
@@ -11,7 +11,7 @@
 
 .build_sysprobe_artifacts:
   - inv -e system-probe.object-files
```

Comment:
why not just removing this line ?

Reply:
Because it's already very hard hard to understand what is building CWS stuff and what is building NPM stuff in those test. I think it would be very unclear if the CWS tests relied on NPM magically generating the object files

Reply:
with -e we know where it is failing, so we could identify the issue CWS or NPM

Reply:
We can do that, I still think we should be more clear about what is building CWS stuff and what is building NPM stuff but this can come in a later PR